noodles-editor/src/app.tsx

```diff
@@ -1,6 +1,26 @@
+import { useEffect, useState } from 'react'
 import TimelineEditor from './timeline-editor'
+import ExamplesPage from './examples-page'
 
 function App() {
+  const [route, setRoute] = useState(window.location.pathname)
+
+  useEffect(() => {
+    const handlePopState = () => {
+      setRoute(window.location.pathname)
+    }
+
+    window.addEventListener('popstate', handlePopState)
+    return () => window.removeEventListener('popstate', handlePopState)
+  }, [])
+
+  // Route: /examples (examples list page)
+  if (route === '/examples' || route === '/examples/') {
+    return <ExamplesPage />
+  }
```

Comment:
This pattern is probably fine for a lightweight router while we don't have many routes. Eventually we could opt into something more declarative like https://github.com/molefrog/wouter